lib/components/card.js

```diff
@@ -80,10 +84,10 @@ export default {
         // We are building up the markup.
         if (slots().img) {
             childNodes.push(slots().img);
-        } else if (props.img) {
+        } else if (props.img && !props.imgBottom) {
             childNodes.push(
                 h(CardImg, {
-                    props: { top: props.imgTop, bottom: props.imgBottom },
+                    props: { top: props.imgTop, bottom: props.imgBottom, imgFluid: props.imgFluid },
```

Comment:
Have a look now. I changed it to accept the img slot as priority, then to dump it up top if top is specified, or if bottom is not specified (making top the default).